src/composables/useNumbers.ts

```diff
@@ -20,6 +20,10 @@ enum PresetFormats {
   percent_lg = '0%'
 }
 
+// Do not display APR values greater than this amount; they are likely to be nonsensical
+// These can arise from pools with extremely low balances (e.g., completed LBPs)
+export const APR_THRESHOLD = 10000;
```

Comment:
useNumbers is a composable for genric number utilities and not specific constants. Could we move this to a relevant /constants file?